src/Foundatio/Queues/QueueStatSummary.cs

```diff
@@ -1,16 +0,0 @@
-﻿using Foundatio.Metrics;
-
-namespace Foundatio.Queues;
-
-public class QueueStatSummary
```

Comment:
This will break a lot of tests, I know in some of our apps we get queue stats and then check counts. Might want to publish a work around in the release notes.

Reply:
So will removing SystemClock and IMetricsClient. It's going to be a breaking change version.